lib/java_buildpack/framework/introscope_agent.rb

```diff
@@ -44,6 +44,8 @@ def release
           .add_system_property('com.wily.introscope.agent.agentName', agent_name(credentials))
           .add_system_property('introscope.agent.defaultProcessName', default_process_name)
 
+        java_opts.add_system_property('agentManager.credential', credential(credentials)) unless
```

Comment:
I'm pretty sure this isn't going to pass `rubocop` (`bundle exec rake` to run static inspection and tests) because of the multiline use of `unless`.  You'll need to rework to satisfy `rubocop`.

Reply:
I originally had "if!" but when I ran rubocop locally, it asked me to use unless instead. 
I changed it to unless, and the next time I used rubocop, it asked me to break the lines, so I did that too. Before I submitted the PR, all tests were passing and rubocop did not mention of any code issues.

Reply:
Never mind, I fixed it. I'll be submitting the updated PR tomorrow.

Reply:
@nebhale 
I have pushed the requested changes.